### PR TITLE
update open video chat schedule

### PR DESCRIPTION
**Pull request by axel-h**

### CODE REVIEW

**Comment by axel-h** [comment]
URL check can't reach https://lixiang.com, seem unrelated.

File: content/contact/home.pml [hunk 1]
```diff
@@ -46,27 +46,19 @@ The next sessions are:
...
-  </li>
-  <li>
-    Topic: <i>open</i>
-    <ul>
-      <li>Sydney: Wed, Nov 17, 8am</li>
-      <li>Central Europe: Tue, Nov 16, 10pm</li>
-      <li>US Pacific Time: Tue, Nov 16, 2pm</li>
+      <li>Sydney: Wed, Dec 1, 8am</li>
+      <li>Central Europe: Tue, Nov 30, 10pm</li>
+      <li>US Pacific Time: Tue, Nov 30, 2pm</li>
```
Comment (lsf37) [inline content/contact/home.pml:53]: I knew I forgot to change the time somewhere. With daylight savings, the US has moved by 1h.

```suggestion
      <li>US Pacific Time: Tue, Nov 30, 1pm</li>
```

File: content/contact/home.pml [hunk 2]
```diff
@@ -46,27 +46,19 @@ The next sessions are:
...
   <li>
-    Topic: <i>open</i>
+    Topics: (open)
     <ul>
-      <li>Sydney: Wed, Dec 1, 8am</li>
-      <li>Central Europe: Tue, Nov 30, 10pm</li>
-      <li>US Pacific Time: Tue, Nov 30, 2pm</li>
+      <li>Sydney: Wed, Dec 15, 8am</li>
+      <li>Central Europe: Tue, Dec 14, 10pm</li>
+      <li>US Pacific Time: Tue, Dec 14, 2pm</li>
```
Comment (lsf37) [inline content/contact/home.pml:61]:
```suggestion
      <li>US Pacific Time: Tue, Dec 14, 1pm</li>
```

**Review by lsf37 — APPROVED** [review]
Thanks for updating, I had promptly forgotten we have the schedule on the website.

**Comment by axel-h** [comment]
Fixed. And adding UTC as reference also.

**Comment by lsf37** [comment]
Thanks!

**Pull request merged**